doc/nrf/links.txt

```diff
@@ -1523,3 +1523,7 @@
 
 .. _`Git for Windows`: https://git-scm.com/download/win
 .. _`nRF Terminal documentation`: https://nrfconnect.github.io/vscode-nrf-connect/terminal/nrfterminal.html
+
+.. _`curl`: https://curl.se/
+
+.. _`nRF54H20 firmware bundle`: https://files.nordicsemi.com/ui/repos/tree/General/SDSC/external/nrf54h20_soc_binaries_v0.3.0.zip
```

Comment:
The link is invalid.

Reply:
This is supposed to be the final link to the final version of the bundle. It does not exist yet